ui/page/seedbackup/save_seed.go

```diff
@@ -90,6 +91,22 @@ func (pg *SaveSeedPage) OnNavigatedTo() {
 				row2 := wordList[11:22]
 				row3 := wordList[22:]
 
+				//for mobile
+				rowm1 := wordList[:17]
+				rowm2 := wordList[17:]
```

Comment:
```suggestion
				rowMobile1 := wordList[:17]
				rowMobile2 := wordList[17:]
```

Reply:
I have change DefualtThemeColors() function with its true purpose, I will return new object of color, it shouldn't change current color because some function in dark mode use it to get the default color